### PR TITLE
chore: Setup Github infra

### PR DESCRIPTION
**Pull request by efirs**

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+name: Docker
+on:
+  push:
+    branches:
+      - main
+      - fixes1
```
Comment (ovaistariq) [inline .github/workflows/docker.yml:6]: Do we need this branch as well?

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  release:
```
Comment (ovaistariq) [inline .github/workflows/docker.yml:7]: Will this publish a production build (i.e. no debug)?

File: .github/workflows/docs-preview.yml
Comment (ovaistariq) [inline .github/workflows/docs-preview.yml:1]: Let's remove the docs

File: .github/workflows/integration.yml [hunk 3]
```diff
@@ -25,22 +26,24 @@ env:
...
+    runs-on: ubuntu-latest
 
     if: github.event_name != 'pull_request' || !contains(github.event.pull_request.labels.*.name, 'no ci')
 
     strategy:
       fail-fast: false
       matrix:
         include:
-          - { name: "PostgreSQL", task: "pg" }
+          #- { name: "PostgreSQL", task: "pg" }
```
Comment (himank) [inline .github/workflows/integration.yml:43]: Let's remove it?

File: .github/workflows/packages.yml [hunk 4]
```diff
@@ -45,13 +45,14 @@ jobs:
...
-    if: >
-      github.event_name != 'pull_request_target' ||
-      (
-        contains(github.event.pull_request.labels.*.name, 'trust') &&
-        !contains(github.event.pull_request.labels.*.name, 'no ci') &&
-        contains(github.event.pull_request.labels.*.name, 'packages')
-      )
+    if: false
+#    if: >
+#      github.event_name != 'pull_request_target' ||
```
Comment (himank) [inline .github/workflows/packages.yml:50]: ^

File: .github/workflows/docker.yml (hunk 2 above)
Comment (efirs) [reply .github/workflows/docker.yml:6]: Yes. `production.Dockerfile` is used below.

**Pull request merged**